lib/tasks/populate_published_groups.rake

```diff
@@ -0,0 +1,18 @@
+desc 'One-time task to populate Tag#published_groups'
+task populate_published_groups: [:environment] do
+  Tag.only_children.each do |tag|
+    item = CollectionsPublisher.services(:content_store).content_item(tag.base_path)
+    unless item
+      puts "#{tag.base_path} is missing from the content-store."
+      next
+    end
+
+    tag.update!(published_groups: item['details']['groups'])
```

Comment:
It would be good for this to handle the case where the content item doesn't have a groups key.  We should still be populating `published_groups` with empty array in this case.